templates/full-time-program.html

```diff
@@ -52,8 +52,9 @@ <h1 class="blue-h1">The Program</h1>
       </div>
       <p class="justify">
         The program usually runs Monday through Friday, 8:00 AM to 5:00 PM (Pacific Time)
-        for six months (from January to June and July to December). The whole program is
-        a year long, starting with training at Techtonica, and followed by six months of
+        for six months (from January to June and July to December). The July 2024 cohort will
+        be a five month cohor, running from July to November. The whole program is
+        about a year long, starting with training at Techtonica, and followed by six months of
```

Comment:
"The whole program is about a year long, starting with training at Techtonica, and followed by six months of" —> "The training is followed by six months of"

Reply:
### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).